ntge-core/src/message/decryptor.rs

```diff
@@ -58,3 +59,16 @@ impl<'a> Decryptor<'a> {
         }
     }
 }
+
+impl<'a> Decryptor<'a> {
+    pub fn verify(public_key: &PublicKey, message: &[u8], signature_bytes: &[u8]) -> bool {
+        let signature: Signature = match Signature::from_bytes(&signature_bytes) {
+            Ok(sig) => sig,
+            Err(_) => return false,
+        };
+        match ed25519::verify(public_key, message, &signature) {
+            Ok(_) => true,
```

Comment:
Since we treat `OK(_)` as a success, how about also change `ed25519:verify` to return `Result<Void, error::CoreError>`?

Reply:
good point.